capstone/src/main/webapp/script.js

```diff
@@ -12,44 +12,105 @@
-  const outputElement = document.getElementById('perspective-output-container');
-  if (!outputElement) {
-    return;
-  }
-  outputElement.innerHTML = '';
-
-  if (toxicityData.attributeScores) {
-    for (let key in toxicityData.attributeScores) {
-      if (toxicityData.attributeScores[key].summaryScore && 
-          toxicityData.attributeScores[key].summaryScore.value) {
-        attributeElement = createAnyElement('p', key + ": " + toxicityData.attributeScores[key].summaryScore.value);
-        outputElement.appendChild(attributeElement);
-      }
+/** Loads the Google Charts API */
+function loadChartsApi(toxicityData) {
+  google.charts.load('current', {'packages':['corechart']});
+  google.charts.setOnLoadCallback(function() {drawBarChart(toxicityData);}); 
+}
+
+/** Draws a Google BarChart from a Perspective JSON. */
+function drawBarChart(toxicityData) {
+  document.getElementById('chart-container').innerHTML = '';
+  const data = google.visualization.arrayToDataTable([[{label: 'Attribute'}, {label: 'Score', type: 'number'}, {role: "style"}]]);
+
+  Object.keys(toxicityData.attributeScores).forEach((attribute) => {
+    var color = '#6B8E23'; // Green
+    const score = toxicityData.attributeScores[attribute].summaryScore.value;
+    if (score >= 0.8) {
+      color = '#DC143C'; // Red
+    } else if (score >= 0.2) {
+      color = '#ffd800'; // Yellow
     }
-  }
+    data.addRow([attribute, score, color]);
+  });
+
+  data.sort({column: 1, desc: false});
+
+  const options = {
+    title: 'Perspective Feedback',
+    bars: 'horizontal',
+    height: 700,
+    legend: {position: "none"},
+    theme: 'material', 
+    hAxis: {viewWindow: {min: 0, max: 1}}
+  };
+
+  const chart = new google.visualization.BarChart(document.getElementById('chart-container'));
+  chart.draw(data, options);
 }
 
-/**
- * Create a 'tag' element with 'text' as its inner HTML
- */
-function createAnyElement(tag, text) {
-  const textElement = document.createElement(tag);
-  textElement.innerHTML = text;
-  return textElement;
+/** Shows the avaiable attributes given a language selected on text analyzer page */
+function showAvailableAttributes() {
+  const langElement = document.getElementById('languageForAnalysis');
+  if (!langElement) {
+    return;
+  }
+  const lang = langElement.value;
+  const avaiableAttributesElement = document.getElementById('available-attributes');
+  avaiableAttributesElement.innerHTML = '';
+	
+  const attributes = ATTRIBUTES_BY_LANGUAGE[lang];
+  attributes.forEach(function(attribute) {
+    const checkbox = document.createElement('input');
+    checkbox.type = "checkbox";
+    checkbox.value = attribute;
+    checkbox.id = attribute + '-checkbox';
+    checkbox.checked = true;
+  
+    const label = document.createElement('label');
+    label.htmlFor = attribute + '-checkbox';
+    label.appendChild(document.createTextNode(attribute));
+  
+    avaiableAttributesElement.appendChild(checkbox);
```

Comment:
nit: I found that adding white space improves the appearance of the check-boxes. If you disagree feel free to leave it as is. 
    avaiableAttributesElement.appendChild(checkbox);
    avaiableAttributesElement.appendChild (document.createTextNode (" "));
    avaiableAttributesElement.appendChild(label);
    avaiableAttributesElement.appendChild (document.createTextNode (" "));

Reply:
Added